src/github.com/oxtoacart/bpool/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
bpool is not new, we already use it in our server proxies.  It's just newly pulled into this workspace.